### ISSUE TITLE
Handle gracefully Calcite exception inside the CLI

### ISSUE THREAD

**Issue by diegosalvi** [diennea/herddb#491]
**FEATURE REQUEST**

1. Please describe the feature you are requesting.
_When executing "broken" queries on CLI we get big stacktraces with not much infos to non developers. We should have better use informations and messages at least on CLI and leave stacktraces to logs (if any... they could go to debug)_
2. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?
_nice-to-have_
3. Provide any additional detail on your proposed use case for this feature.
```
ERROR:java.sql.SQLException: herddb.model.StatementExecutionException: Non-query expression encountered in illegal context
        at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:342)
        at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:445)
        at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:181)
        at herddb.network.netty.NettyChannel.lambda$handlePduRequest$0(NettyChannel.java:96)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by aluccaroni** [comment]
Some other errors examples:

```
herd: CREATE TABLESPACE fagiano¶½
Executing query:CREATE TABLESPACE fagiano¶½
ERROR:java.sql.SQLException: herddb.model.StatementExecutionException: unable to parse query EXECUTE createtablespace fagiano¶½
        at herddb.sql.DDLSQLPlanner.parseStatement(DDLSQLPlanner.java:237)
        at herddb.sql.DDLSQLPlanner.translate(DDLSQLPlanner.java:220)
        at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:244)
        at herddb.server.ServerSideConnectionPeer.handleExecuteStatement(ServerSideConnectionPeer.java:706)
        at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:138)
        at herddb.network.netty.NettyChannel.lambda$handlePduRequest$0(NettyChannel.java:96)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: net.sf.jsqlparser.parser.TokenMgrError: Lexical error at line 1, column 33.  Encountered: "\u00b6" (182), after : ""
        at net.sf.jsqlparser.parser.CCJSqlParserTokenManager.getNextToken(CCJSqlParserTokenManager.java:3060)
        at net.sf.jsqlparser.parser.CCJSqlParser.jj_scan_token(CCJSqlParser.java:16085)
        at net.sf.jsqlparser.parser.CCJSqlParser.jj_3R_131(CCJSqlParser.java:13556)
        at net.sf.jsqlparser.parser.CCJSqlParser.jj_3_78(CCJSqlParser.java:14301)
        at net.sf.jsqlparser.parser.CCJSqlParser.jj_2_78(CCJSqlParser.java:10407)
        at net.sf.jsqlparser.parser.CCJSqlParser.PrimaryExpression(CCJSqlParser.java:6053)
        at net.sf.jsqlparser.parser.CCJSqlParser.BitwiseXor(CCJSqlParser.java:5903)
        at net.sf.jsqlparser.parser.CCJSqlParser.MultiplicativeExpression(CCJSqlParser.java:5859)
        at net.sf.jsqlparser.parser.CCJSqlParser.AdditiveExpression(CCJSqlParser.java:5820)
        at net.sf.jsqlparser.parser.CCJSqlParser.ConcatExpression(CCJSqlParser.java:5753)
        at net.sf.jsqlparser.parser.CCJSqlParser.BitwiseAndOr(CCJSqlParser.java:5777)
        at net.sf.jsqlparser.parser.CCJSqlParser.SimpleExpression(CCJSqlParser.java:5745)
        at net.sf.jsqlparser.parser.CCJSqlParser.SimpleExpressionList(CCJSqlParser.java:5629)
        at net.sf.jsqlparser.parser.CCJSqlParser.Execute(CCJSqlParser.java:7427)
        at net.sf.jsqlparser.parser.CCJSqlParser.SingleStatement(CCJSqlParser.java:182)
        at net.sf.jsqlparser.parser.CCJSqlParser.Statement(CCJSqlParser.java:80)
        at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:52)
        at herddb.sql.DDLSQLPlanner.parseStatement(DDLSQLPlanner.java:235)
        ... 11 more
```

`herddb.model.StatementExecutionException: From line 1, column 15 to line 1, column 21: Object 'MIMMINO' not found at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:343) at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:445) at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:181) at herddb.network.netty.NettyChannel.lambda$handlePduRequest$0(NettyChannel.java:96) at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) at java.base/java.lang.Thread.run(Thread.java:835)`

**Comment by eolivelli** [comment]
@amitvc  what about picking this issue up? 
It has been remarked many times by new users.
We can simply do not print stacktraces of errors and add a -verbose options to print them

**Comment by amitvc** [comment]
Yes I will take this up.

**Comment by eolivelli** [comment]
This isssue has been fixed with #515
